content/blog/2020/06/02/camel-quarkus-release-1.0.0-CR2.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Camel Quarkus 1.0.0-CR2 Released"
+date: 2020-06-02
+authors: ["ppalaga"]
+categories: ["Releases", "Camel Quarkus"]
+preview: What's new in Camel Quarkus 1.0.0-CR2
+---
+
+We are pleased to announce the release 1.0.0-CR2 of Camel Quarkus. Camel Quarkus brings the outstanding integration
+capabilities of Apache Camel to [Quarkus](https://quarkus.io/) - the toolkit for writing subatomically small and
+supersonically fast Java, Kotlin and Scala applications.
+
+So what is new in Camel Quarkus 1.0.0-CR2?
+
+## New bits
+
+While we do not have any new extensions this time, the following extensions were promoted from JVM-only to JVM+native:
+
+* [REST OpenApi](https://camel.apache.org/camel-quarkus/latest/extensions/rest-openapi.html)
+* [Avro data format](https://camel.apache.org/camel-quarkus/latest/extensions/avro.html)
+* [MongoDB GridFS](https://camel.apache.org/camel-quarkus/latest/extensions/mongodb-gridfs.html)
+* [Debezium PostgreSQL](https://camel.apache.org/camel-quarkus/latest/extensions/debezium-mysql.html)
+* [Debezium MySQL](https://camel.apache.org/camel-quarkus/latest/extensions/debezium-mysql.html)
+
+All supported bits can be seen in the [List of Camel Quarkus extensions](https://camel.apache.org/camel-quarkus/latest/list-of-camel-quarkus-extensions.html).
+
+## New documentation pages
+
+Each Camel Quarkus extension has a separate page now. The most of the content is generated from the data available in
```

Comment:
`Most of the content` instead of `The most of the content`

Reply:
Thanks, will change that.